aimsun_scripts/README.md

```diff
@@ -22,10 +22,10 @@ To create and execute the microsimulation in Aimsun, you must have a Aimsun base
 
 Once the Aimsun base network file and input data is available, please follow the following steps:
 
-1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation).
+1. Change `__REPO_PATH` in `append_github_path_to_python.py` to your absolute path for this repository (traffic-microsimulation). Escape any special characters if necessary.

```

Comment:
This is specific to the OS that the user is using. I believe it is a fair assumption that the user can create correct paths for their OS on their own, and thus the addition is not necessary.
Please let me know if you disagree!

Reply:
I agree, it's a more detailed spec for the audience